### PR TITLE
Dgbnoob/digibyte.wiki

### PR DESCRIPTION
**Pull request by DGBNOOB**
I added technical implementations to the terminology file and edited the format (e.g. header, bold, italic and bullet points). I created a biography file for Jared Tate. Included the 0.5% pre-mine details in home file. I think this gives us a good base to update some of the digibyte .org embedded wiki links. Please feel free to edit and add more details if needed.

### CODE REVIEW

File: Contribution-Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+- To participate in discussions and comment on articles, users must be logged in.
+- Users can log in to the wiki platform using their GitHub credentials, simplifying the authentication process.
+
+## Suggestions and Proposals
+
+If you wish to make suggestions or propose new content or changes to existing content, we encourage you to create GitHub Issues on our repository. This helps us manage and track suggestions, making it easier for the community to engage with your proposals.
+
+## Contribution Workflow
+
+1. Fork the [DigiByte Wiki GitHub Repository](https://github.com/dgbat/digibyte-wiki) to your GitHub account.
```
Comment (Sbosvk) [inline Contribution-Guidelines.md:43]: Old link

File: Contribution-Guidelines.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+1. **Markdown (MD)**: Markdown is the preferred format for contributions. It is well-supported by the wiki platform.
+2. **HTML**: Contributions in HTML are accepted but are not the preferred format. Ensure that your HTML is well-structured and compliant.
+3. **Plaintext**: While plaintext contributions are accepted, they should be used sparingly, as they offer limited formatting options.
+
+## Wiki Platform
+
+Our wiki platform runs on Wiki.js, a modern and user-friendly platform for documentation. To ensure your contributions are compatible with our platform, please refer to our [Markdown Guide](https://wiki.digibyte.org/en/contribute/markdown-guide).
+
+## Discussion and Comments
```
Comment (Sbosvk) [inline Contribution-Guidelines.md:30]: This entire section is irrelevant when hosted as GitHub wiki

File: Contribution-Guidelines.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+## Preferred Syntax for Contribution
+
+We accept contributions in the following formats, listed in order of preference:
+
+1. **Markdown (MD)**: Markdown is the preferred format for contributions. It is well-supported by the wiki platform.
+2. **HTML**: Contributions in HTML are accepted but are not the preferred format. Ensure that your HTML is well-structured and compliant.
+3. **Plaintext**: While plaintext contributions are accepted, they should be used sparingly, as they offer limited formatting options.
+
+## Wiki Platform
```
Comment (Sbosvk) [inline Contribution-Guidelines.md:26]: Irrelevant section for GitHub wiki

File: Contribution-Guidelines.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+Your contributions are highly appreciated, and they play a crucial role in maintaining and improving the DigiByte Blockchain Community Wiki. Thank you for being part of our community!
+
+If you have any questions or need assistance with the contribution process, please feel free to reach out to our maintainers or community members.
+
+# Git Contribution Guide
+This guide provides a quick reference for using Git to contribute to the DigiByte Blockchain Community Wiki. It assumes you have a basic understanding of Git.
+
+## Cloning the Repository
+
+1. **Fork the Repository**: Click the <kbd>Fork</kbd> button on the [DigiByte Wiki GitHub Repository](https://github.com/dgbat/digibyte-wiki) to create a copy in your GitHub account.
```
Comment (Sbosvk) [inline Contribution-Guidelines.md:68]: Old link

File: Contribution-Guidelines.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+### Push and Create a Pull Request
+
+1. **Push Your Branch**: Push your branch to your forked repository on GitHub.
+
+```bash
+git push origin feature/descriptive-branch-name
+```
+
+2. **Create a Pull Request (PR)**: Navigate to the [DigiByte Wiki GitHub Repository](https://github.com/DigiByte-Core/digibyte/wiki), click on "New Pull Request," and choose your branch.
```
Comment (Sbosvk) [inline Contribution-Guidelines.md:117]: Old link

**Review by Sbosvk — COMMENTED** [review]
Very interesting to see that despite so many conversations about this, we are now seeing Jared Tate biography re-introduced to a DigiByte wiki.

There are probably more links and other irrelevant info. I mostly wanted to highlight that there are some. I have not made a full review.

**Comment by DGBNOOB** [comment]
@Sbosvk thank you for reviewing. Your comments make sense and good attention to detail. 

The information I added to the wiki files was to have a source of information to link DigiByte .org to GitHub wiki. 

Thank you for pointing out the old links and that the contribution guidelines need to be updated to reflect the correct contribution guidelines. 

I added Jareds bio specifically to have a source to match the W Biography link that is on DigiByte .org. If the reviewers want it out we can remove it and the W Biography link on DigiByte .org. 

I created an issue on DigiByte .org pointing out the embedded links to dgbwiki .com (old wiki), if this PR is approved then we can update the DigiByte .org links to GitHub wiki.

Oliver,
Thank you for taking the time to review, your feedback is always valuable.

[DigiByte .org issue 127](https://github.com/DigiByte-Core/DigiByte.org/issues/127#issue-2473893860)

**Comment by Sbosvk** [comment]
I see. I think considering that there has been discussions about making the website more actual dgb  protocol centric, I assumed things like biographies would also be removed together with socials links. If not,one has to question the real reason for the initiative to clean up the website. I don't see why anyone would care about Jared's life more than any other contributors over the years. Maybe we should write biographies on all of them...you see my point?

**Comment by ycagel** [comment]
I would remove any biographies.

**Comment by DGBNOOB** [comment]
Closing pull request at this time.

**Comment by Sbosvk** [comment]
Why closed? :)

**Comment by DGBNOOB** [comment]
I closed the PR because when I was working on my local repository I was only trying to push the changes I made but all the files were pushed. I’m working on figuring out the git process and also will make edits to the discrepancies you pointed out. Thank you again for your feedback, I went back and read the proposed changes and you’re correct about making the website DigiByte centric.

**Comment by Sbosvk** [comment]
Aha I see. Git can take some time understanding. Sometimes the best way is the quick way. :)

**Pull request closed without merging**